yarn-audit-known-issues

```diff
@@ -1 +1 @@
-{"actions":[],"advisories":{},"muted":[],"metadata":{"vulnerabilities":{"info":0,"low":0,"moderate":0,"high":0,"critical":0},"dependencies":828,"devDependencies":0,"optionalDependencies":0,"totalDependencies":828}}
+{"actions":[],"advisories":{},"muted":[],"metadata":{"vulnerabilities":{"info":0,"low":0,"moderate":0,"high":0,"critical":0},"dependencies":828,"devDependencies":0,"optionalDependencies":0,"totalDependencies":828}}
```

Comment:
New line

Reply:
@ptrelease, this is an auto-generated file, happy to add in newline but does not come with one by default.